src/server/command/debugCommand.ts

```diff
@@ -0,0 +1,103 @@
+import { AnyChannel, Client, Message } from 'discord.js';
+import { prefix } from '../index';
+
+async function getCommand(client: Client, message: Message) {
+  if (message.author.bot || !message.guild) return;
+
+  const command = `${prefix || '!'}debug`;
+  const str = message.content;
+  const match = str.match(command);
+  if (!match) return;
+
+  const commandArgs = str.slice(prefix?.length).trim().split(' ');
+  const argsMessageId = commandArgs[1];
+  if (!argsMessageId) {
+    await message.reply({
+      content: `**メッセージIDが指定されていません。**\n Usage: \`${
+        prefix || '!'
+      }debug <メッセージID>\``
+    });
+    return;
+  }
+
+  const server = await client.guilds.fetch(message.guild.id);
+  const channel = await client.channels.fetch(message.channel.id);
+  if (!server || !channel) {
```

Comment:
この関数の最初で Guild 内のメッセージであることは確認しているので不要かと
```suggestion
  const channel = await client.channels.fetch(message.channel.id);
  if (!channel) {
```